projects/components/src/spotlight-search/spotlight-search.component.html

```diff
@@ -0,0 +1,47 @@
+<clr-modal
+    [(clrModalOpen)]="open"
+    [clrModalStaticBackdrop]="false"
+    (keydown.arrowup)="onArrowUp($event)"
+    (keydown.arrowdown)="onArrowDown($event)"
+    (keydown.enter)="onEnterKey($event)"
+>
+    <div class="modal-body">
+        <div class="search-input-container">
+            <clr-icon shape="search" size="20"></clr-icon>
+            <input
+                #searchInput
+                type="text"
+                class="clr-input"
+                [placeholder]="placeholder || ''"
+                [(ngModel)]="searchCriteria"
+            />
+        </div>
+        <div class="search-result-container">
+            <div *ngFor="let searchSection of searchSections" class="search-result-section">
+                <div
+                    *ngIf="
+                        searchSections.length > 1 &&
+                        (searchSection.isLoading || searchSection.results.length > 0 || showEmptySection)
```

Comment:
This could still show an empty header. I think we need to not show any sections without results. If If it turns out to be without any sections, it needs a placeholder